lib/images/Makefile

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2020 Tasos Alvas <tasos.alvas@qwertyuiopia.com>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# DebOps Logo Makefile
+# See https://docs.debops.org/en/master/meta/logo.html for build requirements
+
+BUILDDIR = .
+SRCDIR = src
+
+SVG_TARGETS = $(patsubst $(SRCDIR)/%-src.svg,$(BUILDDIR)/%.svg,$(wildcard $(SRCDIR)/*-src.svg))
+
+.PHONY: clean all
+all: debops.png debops-text.png debops-small.png favicon.ico $(SVG_TARGETS)
+
+# Optimized SVG: Use these unless forced to do otherwise
+$(BUILDDIR)/%.svg: $(SRCDIR)/%-src.svg
+	svgo $< -o $@
+
+# PNG with text
+debops-text.png: $(SRCDIR)/debops-src.svg
+	inkscape $< -h 200 -o $@
```

Comment:
The horizontal resolution should be 1024 instead of 200 in order to make debops-text.png reproducible.